databricks/koalas/frame.py

```diff
@@ -7964,13 +7960,15 @@ def info(
         ...           encoding="utf-8") as f:
         ...     _ = f.write(s)
         >>> with open('%s/info.txt' % path) as f:
-        ...     f.readlines()  # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
-        [...databricks.koalas.frame.DataFrame...,
+        ...     f.readlines()  # doctest: +SKIP
+        ["<class 'databricks.koalas.frame.DataFrame'>\\n",
```

Comment:
skip this for supporting pandas<1.0.0 & pandas>=1.0.0 both since their behavior are different depend on pandas version.